src/backend/dx12/src/device.rs

```diff
@@ -538,6 +541,9 @@ impl Device {
     ) -> Result<(), image::ViewError> {
         #![allow(non_snake_case)]
 
+        if info.component_mapping != IDENTITY_MAPPING {
+            warn!("Unsupported component_mapping for RTV: {:?}", info.component_mapping);
```

Comment:
I don't think we should warn for these as it's not like this is allowed in the first case for vulkan in any use case.